src/index.tsx

```diff
@@ -30,11 +30,6 @@ import 'tinymce/skins/content/default/content.min.css'
 initializeIcons();
 initializeFileTypeIcons();
 
-tinymce.init({
```

Comment:
Is this not needed?

Reply:
No sir. Because we're using the React component rather than using jQuery to replace everything based on a selector. Other options are set when we include the componentn